lib/discourse_js_processor.rb

```diff
@@ -143,9 +143,9 @@ def babel_source(source, opts = nil)
 
       if opts[:module_name] && !@skip_module
         filename = opts[:filename] || 'unknown'
-        "Babel.transform(#{js_source}, { moduleId: '#{opts[:module_name]}', filename: '#{filename}', ast: false, presets: ['es2015'], plugins: [['transform-modules-amd', {noInterop: true}], 'proposal-object-rest-spread', 'proposal-optional-chaining', ['proposal-decorators', {legacy: true} ], exports.WidgetHbsCompiler] }).code"
+        "Babel.transform(#{js_source}, { moduleId: '#{opts[:module_name]}', filename: '#{filename}', ast: false, presets: ['es2015'], plugins: [['transform-modules-amd', {noInterop: true}], 'proposal-object-rest-spread', 'proposal-optional-chaining', 'proposal-class-properties', ['proposal-decorators', {legacy: true} ], exports.WidgetHbsCompiler] }).code"
```

Comment:
It would be great if these lists of Babel plugins were more readable, it's hard to scan them now for already enabled plugins.

Reply:
Not a bad suggestion, I'll follow up.